docs/tools/sdks/contract-sdks.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Contract SDKs"
+sidebar_position: 10
+---
+
+Contract SDKs are used to build and interact with smart contracts on the Stellar network.
+
+:::note
+
+For Client and XDR SDKs, visit this [page](./library.mdx).
+
+:::
+
+The Rust SDK is maintained by SDF. The AssemblyScript SDK is maintained by dedicated community developers. All SDKs are open-source; file a GitHub issue or pull request in the specific SDK repository if you have questions or suggestions.
+
+Each SDK has its own source code and documentation. Learn how to use a specific SDK by referring to the documentation.
+
+## Soroban Rust SDK
```

Comment:
I think we should add a section below this, `## Soroban Rust Token SDK`.

It is available here: https://docs.rs/soroban-token-sdk, and is also maintained by SDF and contains utilities for building tokens and can be used in addition to the soroban-sdk.

We could add this in a follow up.

cc @sisuresh